boefjes/boefjes/config.py

```diff
@@ -11,21 +11,21 @@ class RuntimeConfiguration(Enum):
 
 class Settings(BaseSettings):
     base_dir: Path = Path(__file__).parent.resolve()
-    log_cfg: Path = Path(__file__).parent / "logging.json"
+    log_cfg: Path = Path(__file__).parent / "logging.json"  # Follow-up ticket to make logging the same for all modules?
```

Comment:
Yes and I think we should also be able to set the log level and maybe other log parameters using environment variables.